templete/pages/signin/index.jsx

```diff
@@ -0,0 +1,6 @@
+import { SignInLayout } from "@/src/page-layout/SignLayout/SignInLayout";
+const SignIn = () => {
+  return <SignInLayout></SignInLayout>;
```

Comment:
컨텐츠가 없으면 self-closing으로 표현해주는게 좋습니다.
```suggestion
  return <SignInLayout />;
```

Reply:
layout이라는 것은 내부에 보여질 컨텐츠와 같이 쓰이는게 의미적으로 알맞을 것 같은데요.
```ts
<Layout>
  <SignInForm />
</Layout>
```

코드에서 처럼 사용되는건 레이아웃이라고 하기엔 멘탈 모델에 맞지 않는 것 같습니다.